modules/generic_worker/templates/generic-worker.config.multiuser.erb

```diff
@@ -0,0 +1,42 @@
+{
+  "accessToken": "<%= @taskcluster_access_token %>",
+  "cachesDir": "<%= @caches_dir %>",
+  "certificate": "",
+  "checkForNewDeploymentEverySecs": 0,
+  "cleanUpTaskDirs": true,
+  "clientId": "<%= @taskcluster_client_id %>",
+  "deploymentId": "",
+  "downloadsDir": "<%= @downloads_dir %>",
+  "idleTimeoutSecs": 345600,
+  "instanceId": "",
+  "instanceType": "",
+  "livelogExecutable": "livelog",
+  "livelogPUTPort": 60022,
+  "numberOfTasksToRun": 1,
```

Comment:
We should change this to `0`. A value of `1` implies you only intend to run one task on the worker, so no new task user will be created for the subsequent task. Setting to 0 implies you intend to run an unlimited number of tasks on this worker.